CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to the COBOL Language Support extension are documented in this file.
 
+## [0.8.3]
+
+- Syntax highlighting for COBOL code enabled.
+- Syntax check for keywords highlights potential errors and mistakes in COBOL code.
+- Autocomplete suggestions  for COBOL keywords while you type
```

Comment:
Please remove the last two points.
About the first point please write something like "Syntax highlighting for variables and paragraphs in COBOL file"
Add the point about java fix